ckanext/switzerland_users/controllers.py

```diff
@@ -51,6 +51,7 @@ def index(self):
         c.pagination = _get_pagination(request, len(users), page_size)
         c.roles = _get_role_selection(c.user, userroles)
         c.organizations = _get_organization_selection(organization_tree, user_admin_organizations)  # noqa
+        c.env = config.get('ckanext.switzerland_users.env')
```

Comment:
Can we please make this safer to fail, by providing a default, if the environment variable is not set.